assets.json

```diff
@@ -0,0 +1,86 @@
+[
+  {
+    "caip-20": "",
+    "iov-name-service-uri": "asset:atom",
```

Comment:
@antoineherzog , do we want to change `iov-name-service-uri` to `starname-uri` or simply `uri`?

Reply:
Lets change to starname-uri just to be explicit for developers I think